nautobot_golden_config/models.py

```diff
@@ -213,19 +213,31 @@ class ComplianceRule(PrimaryModel):  # pylint: disable=too-many-ancestors
         help_text="Whether or not the configuration order matters, such as in ACLs.",
     )
     match_config = models.TextField(
-        null=True,
-        blank=True,
+        null=False,
```

Comment:
Shouldn't this generate a migration? 

Also, I think we left off with the idea that we would allow for `*` for all.

Reply:
Good catch!

I re-thought this , and having blank in my opinion is reasonable as a "match all" :

- it is already handled (denied) in the code for CLI
- it is backwards compatible with JSON, it does not require to change users' data
- it does not require to change default behaviours
- we could add a support in the future for CLI , that blank would mean "match everything in the config"